examples/ota-requestor-app/esp32/sdkconfig.defaults

```diff
@@ -55,3 +55,4 @@ CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
 
 # Enable OTA Requestor
 CONFIG_ENABLE_OTA_REQUESTOR=y
+CONFIG_DEVICE_SOFTWARE_VERSION_NUMBER=5
```

Comment:
Why is the value kept as 5? Is it required to be set explicitly? Or you have added it so that application can set it to a different number for the upgraded firmware?

Reply:
It is just an example how this can be used. If not set explicitly default is 1. Ideally when upgrading the firmware this shall be bumped and that firmware should be used for OTA.

Reply:
If it is to be bumped up, should we set it to 2?

Reply:
Changed to 2.